_pages/privacy.md

```diff
@@ -2,10 +2,22 @@
+<p>The GSA Privacy Office preserves and evolves privacy protections for any individual whose personal information is handled by the agency. In practice, this requires the Office to conduct a thorough, periodic analysis of each of the GSA&rsquo;s programs and information systems.</p>
 
-* [Privacy Impact Assessment (PIA)](pia/)
-* [Personally Identifiable Information (PII)](pii/)
-* [System of Records Notice (SORN)](sorn/)
+<p>As someone interested in shipping a product or service, you&rsquo;ll need to first consider <strong>the agency</strong> at which the product or service will be authorized to operate. If that agency is someone <i>other</i> than GSA, please stop reading this guide and contact the Privacy Office at the authorizing agency. Otherwise, take the following steps:</p>
 
-For help, reach out to [#wg-privacy](https://gsa-tts.slack.com/messages/C4F2HMGBE/).
+<ol>
+  <li>
+    <a href="https://gsa.gov/portal/getMediaData?mediaId=199855">GSA Policy 1878.2A CIO P Conducting Privacy Impact Assessments (PIAs)</a> requires that each of the GSA&rsquo;s information systems, either planned or under development, be subject to an annual privacy threshold analysis. Please <a href="https://docs.google.com/a/gsa.gov/forms/d/e/1FAIpQLScrhrrzgS-ZcOpS37YzLdmVXkwX_3dBefZ1t39oKcgdVepmuQ/viewform">fill out and submit this Google Form</a> to get started.
+  </li>
+  <li>
+    <p>Once they&rsquo;ve had a chance to review your information system, GSA&rsquo;s Privacy Office will reach out to help you collaboratively identify, discuss, and evaluate privacy risks, and to determine the ways in which they might be mitigated. Generally speaking, a system&rsquo;s privacy risk stems from to the degree to which that system and its associated business processes collect and use personally identifiable information (PII).</p>
+    <p>It&rsquo;s important to note that the Google Form mentioned in step one is designed to prompt a conversation, and that this conversation may ultimate affect the design of the product or service in question. This conversation might also result in the creation of the following privacy-related documentation:</p>
+    <ul>
+      <li><a href="https://docs.google.com/a/gsa.gov/document/d/1CcVLPNNra1WCGqHewK2ojQ_ysHcGxmJ1IlsCo9pAiSU/edit?usp=drive_web">Privacy Act Notices</a></li>
+      <li><a href="https://docs.google.com/document/d/1hDIDvcTv-zplMBl_uQbg7GQ5uwrC-2Jr8xg4D073q5Y/edit">System of Records Notices (SORNs)</a></li>
+      <li><a href="https://docs.google.com/document/d/1o6FtuHPW41ZLkjXnANjZ60-sUojDexh3r3afm1yE9b8/edit#heading=h.s85zskw81fkz">Privacy Impact Assessments (PIAs)</a></li>
+    </ul>
+  </li>
+  <li>As you complete and receive sign off on your product or service&rsquo;s privacy-related documentation, the Privacy Office may work with you to plan for the creation of any necessary role-based training for the staff who will operate the product or service. This ensures privacy compliance in perpetuity.</li>
+</ol>
```

Comment:
Can we do this in Markdown instead, for easier editing and consistency with the rest of the repository?